tools/EntityFramework.settings.targets

```diff
@@ -24,7 +24,8 @@ Project global pre-settings.
     <VersionReleasePrefix>0</VersionReleasePrefix>
     <VersionRelease>$(VersionReleasePrefix)-beta2</VersionRelease>
     <VersionReleaseName>Beta 2</VersionReleaseName>
-    <ToolingDotNetFrameworkVersion Condition="'$(VisualStudioVersion)' == '11.0' OR '$(VisualStudioVersion)' == '12.0' OR '$(VisualStudioVersion)' == '14.0'">v4.5</ToolingDotNetFrameworkVersion>
+    <ToolingDotNetFrameworkVersion Condition="'$(VisualStudioVersion)' == '11.0' OR '$(VisualStudioVersion)' == '12.0'">v4.5</ToolingDotNetFrameworkVersion>
+    <ToolingDotNetFrameworkVersion Condition="'$(VisualStudioVersion)' == '14.0'">4.5.1</ToolingDotNetFrameworkVersion>
```

Comment:
a higher .net version (4.6, 4.7) replaces the older one, so VS2015 RTM comes with .net 4.6 RTM (4.6.81.0) which replaces all 4.5.x releases. It was just a small hint when I looked at this PR.